This member is not inlined by gcc7.2.0 and thus building icecat
fails with an error "function not considered for inlining".

--- dom/bindings/BindingUtils.h	2017-09-01 22:35:55.000000000 +0200
+++ dom/bindings/BindingUtils.h	2017-09-04 10:28:13.794397024 +0200
@@ -246,7 +246,7 @@
 // T* can be assigned to.
 namespace binding_detail {
 template <class T, bool mayBeWrapper, typename U, typename V>
-MOZ_ALWAYS_INLINE nsresult
+nsresult
 UnwrapObjectInternal(V& obj, U& value, prototypes::ID protoID,
                      uint32_t protoDepth)
 {
